File: consumer/src/main/java/ru/romanow/protocols/consumer/web/PingController.java
package ru.romanow.protocols.consumer.web;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.http.MediaType;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import ru.romanow.protocols.api.model.PingResponse;

@RestController
@RequestMapping("/api/ping")
public class PingController {
    private static final Logger logger = LoggerFactory.getLogger(PingController.class);

    @GetMapping(produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
    public PingResponse ping() {
        logger.info("Request to '/cookies'");
        return new PingResponse("ok");
    }
}

File: producer/src/main/java/ru/romanow/protocols/producer/web/PingController.kt
package ru.romanow.protocols.producer.web

import org.slf4j.LoggerFactory
import org.springframework.http.MediaType
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import ru.romanow.protocols.api.model.PingResponse

@RestController
@RequestMapping("/api/v1/ping")
class PingController {
    private val logger = LoggerFactory.getLogger(PingController::class.java)

    @GetMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
    fun ping(): PingResponse {
        logger.info("Request to '/api/v1/ping'")
        return PingResponse("OK")
    }
}